vktrace/vktrace_viewer/vktraceviewer_qtracefileloader.cpp

```diff
@@ -217,7 +218,18 @@ bool vktraceviewer_QTraceFileLoader::populate_trace_file_info(vktraceviewer_trac
         pTraceFileInfo->packetCount++;
         fileOffset += packetSize;
 
-        fseek(pTraceFileInfo->pFile, fileOffset, SEEK_SET);
+        seekResult = fseek(pTraceFileInfo->pFile, packetSize - sizeof(uint64_t), SEEK_CUR);
```

Comment:
We don't need these changes as part of trim, but this change did fix issues that were preventing some trace files from loading in vktraceviewer. I found the issue while testing trim, but it shouldn't be specific to trim.